onnx/backend/test/cpp/driver/utils.h

```diff
@@ -0,0 +1,21 @@
+#include <sstream>
```

Comment:
Seems this file is pretty useful for our users, shall we move it out of `test/cpp/driver`? to some onnxifi_utils.h/cc?

Reply:
@houseroad Yes I do agree with you. Just wondering where should it be moved to.

Reply:
How about the folder as onnxifi.h? which is onnx/

Reply:
Sounds decent for now. At last I guess most of the onnxifi code in there will need to be migrated to somewhere else, since there are too many onnxifi files already...